report-processor/src/main/resources/application.sql.yml

```
@@ -1,4 +1,136 @@
+        coalesce(e.claim3_category, ''),
+        coalesce(e.claim4_category, ''),
+
+        st.gender_code as student_gender_code,
+
+        (case when e.migrant_status then 'Yes' else 'No' end),
+        (case when e.section504 then 'Yes' else 'No' end),
+        (case when e.iep then 'Yes' else 'No' end),
+        (case when e.economic_disadvantage then 'Yes' else 'No' end),
+        (case when e.lep then 'Yes' else 'No' end),
+
+        (case when exists (select 1 from student_ethnicity where student_id = st.id and ethnicity_code = 'HispanicOrLatinoEthnicity') then 'Yes' else 'No' end ),
+        (case when exists (select 1 from student_ethnicity where student_id = st.id and ethnicity_code = 'AmericanIndianOrAlaskaNative') then 'Yes' else 'No' end ),
+        (case when exists (select 1 from student_ethnicity where student_id = st.id and ethnicity_code = 'Asian') then 'Yes' else 'No' end ),
+        (case when exists (select 1 from student_ethnicity where student_id = st.id and ethnicity_code = 'BlackOrAfricanAmerican') then 'Yes' else 'No' end ),
+        (case when exists (select 1 from student_ethnicity where student_id = st.id and ethnicity_code = 'White') then 'Yes' else 'No' end ),
+        (case when exists (select 1 from student_ethnicity where student_id = st.id and ethnicity_code = 'NativeHawaiianOrOtherPacificIslander') then 'Yes' else 'No' end ),
+        (case when exists (select 1 from student_ethnicity where student_id = st.id and ethnicity_code = 'TwoOrMoreRaces') then 'Yes' else 'No' end ),
+        (case when exists (select 1 from student_ethnicity where student_id = st.id and ethnicity_code = 'Filipino') then 'Yes' else 'No' end ),
+
+        coalesce((select GROUP_CONCAT(a.code SEPARATOR '|')
+                  from exam_available_accommodation eaa
+                    join accommodation a on a.id = eaa.accommodation_id
+                  where eaa.exam_id = e.id), '')
+
+      into outfile ${app.export.destination} :export_location
```

This is that export destination I referenced earlier.  So in order to have a local / s3 implementation, we need some way to to set the `S3` aurora command dynamically when we're sending to S3.  When it's local file, it should just be `''`.  I'm looking for this configuration at startup so it will fail fast if it's not configured correctly. I  It was suggested I try this as a property, which works, but I'm open to other options as well.